vars/commonPipelineEnvironment.groovy

```diff
@@ -21,6 +21,8 @@ class commonPipelineEnvironment implements Serializable {
 
     private String mtarFilePath
 
+    String transportRequestId
```

Comment:
for (most?) properties a getter and setter is created, like gitCommitId ->setGitCommitId/getGitCommitId
any convention?

Reply:
getter and setter are automagically created by groovy. There is no need for writing the getters and setters down. See [here](http://groovy-lang.org/style-guide.html#_getters_and_setters).

I personally think each line we have we should be able to explain. How should we explain a written-down getters and setters in the light of groovy generating the getters and setters for you?

Reply:
yes, so why do we have so many getters and setter?